frontend/src/app/modules/availability/new-slot/questions/questions.component.sass

```diff
@@ -4,11 +4,12 @@
 $container-padding-top-bottom: 33px
 $question-input-width: 337px
 
-.bold
-    font-weight: 700
+.title
+    color: $black
+    margin-bottom: 40px
 
 .outside-label
-    margin-bottom: 20px
+    font-weight: 700
```

Comment:
```suggestion
```

Repeating _css_ properties in multiple classes should be spit into modular pieces, if possible. If you don't like using `.bold` class explicitly in `html` elements, please, factor out `font-weight:700` into separate place and comma-enumerate all classes/elements, which need it.